sql/core/src/main/scala/org/apache/spark/sql/catalyst/analysis/ResolveSessionCatalog.scala

```diff
@@ -282,6 +282,10 @@ class ResolveSessionCatalog(
       AlterTableRecoverPartitionsCommand(
         v1TableName.asTableIdentifier,
         "MSCK REPAIR TABLE")
+
+    case RefreshTableStatement(tableName) =>
+      val v1TableName = parseV1Table(tableName, "REFRESH TABLE")
```

Comment:
We have `TableCatalog.invalidateTable` to implement a v2 REFRESH TABLE command.

Reply:
Followed by `TableCatalog.loadTable`? Or`TableCatalog.loadTable` is called lazily (you don't call it inside `RefreshTableExec` - in this case, same behavior as `UNCACHE TABLE`)?

Reply:
Not followed by `loadTable`. It should invalidate and load the table lazily.

Reply:
Thanks. I adde v2 command in this PR.